docs/source/user-guide/sql/scalar_functions.md

```diff
@@ -2663,6 +2695,18 @@ _Alias of [array_element](#array_element)._
 
 _Alias of [array_element](#array_element)._
 
+### `list_has`
+
+_Alias of [array_has](#array_has)._
+
+### `list_has_all`
+
+_Alias of [array_has_a;;](#array_has_all)._
+
+### `list_has_any`
+
+_Alias of [array_has_any](#array_has_any)._
```

Comment:
Also add links to them up here

https://github.com/apache/arrow-datafusion/blob/ff7dfc3787b46715fce2b4e5aa805bb9afb9ce3a/docs/source/user-guide/sql/scalar_functions.md?plain=1#L1731-L1791

Reply:
Sorry, what do you mean?

Reply:
As in add to that list there:

```md
 - [list_has](#list_has)
 - [list_has_all](#list_has_all)
 - [list_has_any](#list_has_any) 
```

So the aliases appear in that list and are linked to